packages/nimble-tokens/package.json

```diff
@@ -19,6 +14,37 @@
     "invoke-publish:nuget": "cross-env-shell dotnet nuget push \"NimbleTokens/bin/Release/*.nupkg\" -k $NUGET_SECRET_TOKEN -s \"https://api.nuget.org/v3/index.json\"",
     "invoke-publish:npm": "npm publish"
   },
+  "wireit": {
```

Comment:
Same comment as elsewhere about responding to changes to `package.json`, `node_modules`, and `tsconfig.json`.

Reply:
wireit automatically tracks changes to lock files: https://github.com/google/wireit#package-locks

So I think that covers the changes that are interesting from package.json / node_modules
The tsconfig is probably useful for the commands that invoke typescript (explicit tsc calls, angular / storybook build)

Reply:
I've added `angular.json` and `tsconfig.json` to the inputs for `build:library` and `build:application`.

Reply:
The build is currently failing, I think because of this change (looks like it's accidentally listed it as a dependency instead of an input).

Reply:
The build is working now for me after your last push.